open-api/swagger.yaml

```diff
@@ -3004,11 +3013,11 @@ definitions:
             type: number
           max:
             type: number
-          average:
+          avg:
             type: number
           count:
             type: number
-          std_dev:
+          stdDev:
```

Comment:
is changing gb covered? is there ticket/PR?

Reply:
Interestingly, the REST API never did anything else, but both this document and Glowing Bear had it wrong. Glowing Bear does not use these aggregates, by the way.